### PR TITLE
Group tests into sections.

### PR DESCRIPTION
**Pull request by BigBlueHat**
It was getting very hard to understand the growing list of MUSTs relative to
the specification. The tests are now grouped into similarly named sections
with links in the code (at least) to those sections.

I plan to do more (likely in the upstream reporter) to make it possible to
link to the sections of the spec and provide basic descriptions of what the
tests are about.

Note: this one stacks on top of #73 which is still being polished.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "chai": "^4.3.6",
     "klona": "^2.0.5",
     "mocha": "^10.0.0",
-    "vc-test-suite-implementations": "github:w3c/vc-test-suite-implementations"
+    "vc-test-suite-implementations": "../vc-test-suite-implementations"
```
Comment (aljones15) [inline package.json:48]: you keep using a local version of vc-test-suite-implementations. have you tried `npm link`?

File: tests/10-vcdm2.js [hunk 2]
```diff
@@ -443,12 +496,40 @@ describe('Verifiable Credentials Data Model v2.0', function() {
...
+    });
+  }
+});
+
+// 4.9 Securing Mechanisms https://w3c.github.io/vc-data-model/#securing-mechanisms
+describe('Securing Mechanisms', function() {
+  setupMatrix.call(this);
+  for(const [name, implementation] of match) {
+    const endpoints = new TestEndpoints({implementation, tag});
+    const createOptions = {challenge};
```
Comment (aljones15) [inline tests/10-vcdm2.js:508]: looks like something changed and the challenge is no longer being used here?

**Review by aljones15 — APPROVED** [review]
I like the way the report reads and this improves test clarity while also restoring the its so definitely approved.

File: package.json (hunk 1 above)
Comment (BigBlueHat) [reply package.json:48]: Sorry that keeps sneaking in. I'll give `npm link` a look--and clean this out again.

File: tests/10-vcdm2.js (hunk 2 above)
Comment (BigBlueHat) [reply tests/10-vcdm2.js:508]: Fixed in 5d6be7b. Sorry I missed those!

**Pull request merged**